### DISCUSSION TITLE
Hi it would be greate to have Contrubuters folder. so we can pr our proof files maybe?

### DISCUSSION THREAD

**Discussion by SonOfBTC** [nexus-xyz/nexus-zkvm#177] [Ideas]
wouldnt it good for contrubuters?

**Comment by sjudson** [comment]
Could you provide more detail on what you mean by 'pr our proof files'?

There are some interesting applications of a 'proof registry', but that would be independent of our core codebase.

**Reply by SonOfBTC** [reply]
In the last step of the repo, there is a verify your proof step. At the end of the process, I received a proof file. I'm talking about adding this as a participant. This way, you can see who is supporting and trying your project.

**Reply by sjudson** [reply]
Proofs are designed for distribution to verifiers in an application-specific way, and although as noted there are some interesting applications of a public 'proof registry', it would necessarily be independent of our core codebase.

**Comment by sjudson** [comment]
Will close discussion, given lack of clarity + any followup/activity.
